packages/manager/modules/cloud-connect/src/overview/datacenter/remove-datacenter/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Don't know if it is provided content, but you can use `datacentre/datacenter` instead of using its full translation `centre de données`. At least to keep consistency with terms use in the whole control panel.

Reply:
We still have not received content. That was google translate :)